imports/plugins/included/payments-paystack/client/settings/containers/PaystackSettingsFormContainer.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { composeWithTracker } from "@reactioncommerce/reaction-components";
+import { Meteor } from "meteor/meteor";
+import { Packages } from "/lib/collections";
+import { TranslationProvider } from "/imports/plugins/core/ui/client/providers";
+import { Reaction, i18next } from "/client/api";
+import { PaystackSettingsForm } from "../components";
+
+class PaystackSettingsFormContainer extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      secretKey: "278302390293",
```

Comment:
it's not a good practice to include application settings and secret keys in your codebase. Let's set these as environment variables